src/Collect/Support/alias.php

```diff
@@ -1,35 +1,20 @@
 <?php
 
 $aliases = [
```

Comment:
Looks like we're missing all the contracts

Reply:
And traits. All added.